vision/cloud-client/detect.py

```diff
@@ -356,7 +356,7 @@ def detect_crop_hints(path):
         content = image_file.read()
     image = vision_client.image(content=content)
 
-    hints = image.detect_crop_hints()
+    hints = image.detect_crop_hints({1.77})
```

Comment:
I agree, this is specific to the hand-written veneers. The API takes an opaque settings object ([ImageContext](https://cloud.google.com/vision/docs/reference/rest/v1/images/annotate#ImageContext)) which is wrapped at the client library layer and is currently missing certain features.